### DISCUSSION TITLE
Acceptance Angle

### DISCUSSION THREAD

**Discussion by axelrannou** [OpenGATE/opengate#426] [Technical issues]
Hello, 

I was wondering why we can only use the [SkipEvent policy with an iso direction type](https://github.com/OpenGATE/opengate/blob/f20223d20cd89530c40a2d5d300ba3551455cc88/core/opengate_core/opengate_lib/GateAcceptanceAngleTesterManager.cpp#L42C1-L47C4).
Because I'm using this histogram source and applying an acceptance angle to make the shape conic
![image](https://github.com/OpenGATE/opengate/assets/92580175/20af7077-452a-4b60-a74a-328b041f697d)
But I don't want to create the particles outside the acceptance angle with 0 energy, I want to have all the particles 
generated inside the acceptance angle.
Is there a reason to have the ShipEvent policy only for iso direction type ?

**Comment by dsarrut** [comment]
It is still an issue with the current version ?

**Reply by axelrannou** [reply]
It was fixed a long time ago, forgot to close the discussion.
